website/OpenEXRFileLayout.rst

```diff
@@ -991,177 +991,858 @@ pixels. The image has two channels: G, of type ``HALF``, and Z, of type
-    pixel data for Z channel                                                  |
-                                                                              |
-.. code-block::
-       
-     00   00   00   18   00   00   00   37   38   76   33   74   3b   73   38   7f
-        1         |         24        |  0.527  |  0.233  |  0.932  |  0.556  |
-        y         |  pixel data size  |  pixel data for G channel             |
-      scan line 1
-    
-.. code-block::
-       
-     ab   e8   3e   8a   cf   54   3f   5b   6c   11   3f   20   35   50   3d   02
-    0.454433      |   0.831292        |   0.56806         |   0.0508319       |
-    pixel data for Z channel                                                  |
-                                                                              |
-    
-.. code-block::
-       
-     00   00   00   18   00   00   00   23   3a   0a   34   02   3b   5d   3b   38
-        2         |         24        |  0.767  |  0.252  |  0.876  |  0.920  |
-        y         |  pixel data size  |  pixel data for G channel             |
-      scan line 2
-    
-.. code-block::
-       
-     f3   9a   3c   4d   ad   98   3e   1c   14   08   3f   4c   f3   03   3f
-    0.0189148     |   0.298197        |   0.531557        |   0.515431       
-    pixel data for Z channel                                                 
-                                                                  end of file
+The first column of the table below lists all the bytes of the file in hexadecimal 
+notation. The second column of the table shows how the bytes are grouped into 
+integers, floating-point numbers and text strings. The third column of the table
+indicate how those basic objects form compound objects such as attributes 
+or the line offset table.
+
+Download the :download:`sample.exr <downloads/sample.exr>`.
+
+.. table::
+  :width: 50%
+
+  +-----------+------------+-------------+
+  |byte       |value       |description  |
+  +===========+============+=============+
+  |76         |20000630    |magic number |
+  +-----------+            |             |
+  |2f         |            |             |
+  +-----------+            |             |
+  |31         |            |             |
+  +-----------+            |             |
+  |01         |            |             |
+  +-----------+------------+-------------+
+  |02         |2           |version,     |
+  +-----------+            |flags        |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+------------+-------------+
+  | Start of header                      |
+  +-----------+------------+-------------+
+  |63         |c           |attribute    |
+  +-----------+------------+name:        |
+  |68         |h           |channels     |
```

Comment:
In this context, I think it would be more appropriate to format the name in fixed type, i.e. `channels` since it's a literal. Here and in the instances of other attributes below.